R/relativeimportance.R

```diff
@@ -279,13 +283,18 @@ subsetDataWeightsAndFormula <- function(formula, data, weights)
 #' @importFrom flipU OutcomeName
 #' @importFrom stats terms.formula
 #' @noRd
-computeJaccardCoefficients <- function(formula, data = NULL, weights, variable.names = NA)
+computeJaccardCoefficients <- function(formula, data = NULL, weights, variable.names = NA,
+                                       outlier.prop.to.remove = 0)
 {
     processed.data <- subsetDataWeightsAndFormula(formula, data, weights)
     relevant.data <- processed.data$data
     weights <- processed.data$weights
     formula <- processed.data$formula
 
+    if (!"non.outlier.data_GQ9KqD7YOf" %in% names(data) &&
+        !is.null(outlier.prop.to.remove) && outlier.prop.to.remove > 0)
+        relevant.data <- removeOutliersFromLinearRegression()
```

Comment:
`removeDataThatAreOutliersFromLinearRegression(formula, relevant.data, weights,
                                                                       outlier.prop.to.remove = outlier.prop.to.remove)`

Reply:
Done in https://github.com/Displayr/flipRegression/pull/44/commits/affc993cef376b95cd581e6c613aba41416ef249 and unit test added. Thanks